cocos/2d/assets/sprite-frame.ts

```diff
@@ -262,7 +266,7 @@ export class SpriteFrame extends Asset {
      * @en uv update event.
      * @zh uv 更新事件。
      */
-    public static EVENT_UV_UPDATED = 'uv_updated';
+    public static EVENT_UV_UPDATED = SpriteFrameEvent.UV_UPDATED;
```

Comment:
We only inline enums, doesn't support to inline static const varibles in class.
So define a enum for those constants.